### PR TITLE
MINOR: fix broken JavaDoc links

### PR DESCRIPTION
**Pull request by mjsax**
Fix broken JavaDoc markup to resolve build warnings.

### CODE REVIEW

File: server-common/src/main/java/org/apache/kafka/queue/EventQueue.java [hunk 1]
```diff
@@ -55,10 +55,9 @@ public FailureLoggingEvent(Logger log) {
         @Override
         public void handleException(Throwable e) {
             if (e instanceof RejectedExecutionException) {
-                log.info("Not processing {} because the event queue is closed.",
-                    this.toString());
+                log.info("Not processing {} because the event queue is closed.", this);
```
Comment (mjsax) [inline server-common/src/main/java/org/apache/kafka/queue/EventQueue.java:58]: Side cleanup: calling `toString()` is redundant.

File: server-common/src/main/java/org/apache/kafka/queue/EventQueue.java [hunk 2]
```diff
@@ -182,7 +181,7 @@ default void scheduleDeferred(String tag,
     enum EventInsertionType {
         PREPEND,
         APPEND,
-        DEFERRED;
+        DEFERRED
```
Comment (mjsax) [inline server-common/src/main/java/org/apache/kafka/queue/EventQueue.java:184]: side cleanup

**Review by dajac — APPROVED** [review]
LGTM

**Review by showuon — APPROVED** [review]
LGTM!

**Pull request merged**